client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -5,7 +5,10 @@ import styled from '@emotion/styled';
 import {
 	__experimentalToggleGroupControl as ToggleGroupControl,
 	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+	Tooltip,
 } from '@wordpress/components';
+import { ToggleGroupControlOptionProps } from '@wordpress/components/build-types/toggle-group-control/types';
+import { hasTranslation } from '@wordpress/i18n';
```

Comment:
In #89239, I used `useI18n()` from `@automattic/i18n-utils`, which exposes a `hasTranslation` function. I _think_ this has the same benefit that `useLocale` has over importing `i18n` directly–it's aware of the i18n context within the React app. 

However, I see now that it might be even better to use `useHasEnTranslation` from `@automattic/i18n-utils`, since that hook kills two birds with one stone: it checks if the active locale is English (including its variations), and calls `hasTranslation`.

Reply:
Great idea